packages/bot-engine/src/services/api/trading-times.js

```diff
@@ -15,8 +15,8 @@ export default class TradingTimes {
         }
 
         this.is_initialised     = true;
-        const { server_time }   = this.root_store.core.common;
-        this.server_time        = server_time.clone();
+        // why clone here?
+        // this.server_time        = server_time.clone();
```

Comment:
This is done across all our apps since Binary static, it's done to ensure the original server time object remains untouched.